assets/source/setup-guide/app/helpers/documentation-link-props.js

```diff
@@ -53,7 +67,7 @@ function documentationLinkProps( {
 				onClick( event );
 			}
 			if ( ! event.defaultPrevented ) {
-				recordEvent( 'pfw_documentation_link_click', {
+				recordEvent( `pfw_${ eventName }`, {
```

Comment:
adding the prefix here is better!

Reply:
But naming convention for Track Events state:

> Don’t programmatically create event names. They should be hardcoded so they are easily searchable in the code base.

Reply:
Fixed here https://github.com/woocommerce/pinterest-for-woocommerce/pull/299/commits/4806fb3f592054a5e2683f7208b1f6e0efaabed3